runtime/go.mod

```diff
@@ -3,25 +3,40 @@ module encore.dev
 go 1.18
 
 require (
-	github.com/felixge/httpsnoop v1.0.2
+	github.com/felixge/httpsnoop v1.0.1
```

Comment:
It looks like we've done some downgrades in here; we should stick to the newer versions of `httpsnoop` and `pgx`, `pgconn` & `pgproto3`